go.mod

```
@@ -104,7 +104,7 @@ replace (
 	github.com/Azure/go-autorest => github.com/Azure/go-autorest v10.14.0+incompatible
 	// Needed due to to Sirupsen/sirupsen case clash
 	github.com/Sirupsen/logrus => github.com/sirupsen/logrus v1.4.2
-	github.com/aws/aws-sdk-go => ./lib/github.com/aws/aws-sdk-go
+	github.com/aws/aws-sdk-go => github.com/weaveworks-experiments/aws-sdk-go v1.25.36
```

Looks like the `weaveworks-experiments` fork uses the `master` branch of my fork, however, the changes are in the `release` branch. You can see that the generated mock files have changed. But that doesn't seem to have affected the build because it looks like `master` does have the changes we're using.

---

So, are you saying we need this? (or so)
```suggestion
	github.com/aws/aws-sdk-go => github.com/weaveworks-experiments/aws-sdk-go b6a927b24aa95dd32b0d7815b5ec2600f609c218
```

---

Can you try `v1.25.14-0.20191118174145-b6a927b24aa9`? It's what I used for my fork.

---

Indeed, `go get`ting did change it to `v1.25.14-0.20191118174145-b6a927b24aa9`.
Will use that everywhere.

---

Done.